src/content/documentation/publish/distribute-manifest-versions.md

```diff
@@ -0,0 +1,110 @@
+
+When deciding which version is most appropriate to distribute to your users through AMO, consider that:
+* When Firefox is updated in early 2023, it may take several weeks for most users to update from a version of the browser that only supports MV2. 
+* Older clients, such as Firefox ESR – the extended support release for enterprises (large companies and organizations) – will remain in use for several months (Firefox ESR 102.x is supported until September 2023 when the ESR release will move to a version supporting MV3).
+
+So, if your user base is large, or you have requirements to provide support for older Firefox versions for an extended period, we recommend that you consider using this alternate distribution for your MV3 version.
+
+If you choose to remain on MV2 and wait to transition to MV3 later, there are steps you can take in MV2 that [move your extension closer to what is necessary in MV3](https://blog.mozilla.org/addons/2022/10/31/begin-your-mv3-migration-by-implementing-new-features-today/). As usual, if your extension depends on features only available to recent Firefox versions, specify the compatible Firefox version in the [`strict_min_version`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_specific_settings) field in manifest.json. Older Firefox extensions will not receive an update, and a new user on an old Firefox version can install an older version of the extension using the “See all versions” link at the extension listing on AMO.
+
+{% endcapture %}
+{% include modules/column-w-toc.liquid,
+  id: "transfer-ownership"
+  content: content_with_toc
+%}
+
+<!-- END: Content with Table of Contents -->
+<!-- Single Column Body Module -->
+
+{% capture content %}
+
+## How to sign an MV3 version
+
+To get a signed version of your MV3 extension:
+
+1. On AMO, when you upload the MV3 extension version, change “Where to host version”  to “on your own”.
+   ::: note
+   You need to change it back to “on this site” for MV2 versions, as the most recently used channel is remembered.
+   :::
+2. For API uploads, specify `channel=unlisted` for the self-distribution channel.
+
+See [Self distribution](/documentation/publish/submitting-an-add-on/#self-distribution), for more details on the process.
+
+{% endcapture %}
+{% include modules/one-column.liquid,
+  id: "listing-on-amo"
+  content: content
+%}
+
+<!-- END: Single Column Body Module -->
+<!-- Single Column Body Module -->
+
+{% capture content %}
+
+## How to distribute the MV3 version
+
+Self-distributed versions cannot be downloaded by users directly from AMO. You must distribute the experimental XPI to users yourself, such as by a download from your website, email, etc.
+
+Remember that:
+* When you distribute the XPI file from a website that is not AMO, you also need to add the `install_origins` key to the manifest. This key lists the origin of the host that you place the XPI on.
```

Comment:
It's weird to see that in a "Remember that" section out of the blue, since it's a new requirement. IMHO at the very least we need to:
1. Mention that it's a new, MV3-only requirement
2. Write an MDN documentation page about `install_origins` (it doesn't exist yet) and link to that page here

Reply:
@rebloor let's get rid of this paragraph